src/popup/components/Tokens.vue

```diff
@@ -137,13 +141,26 @@ export default defineComponent({
         ? mapToken(props.tokens[2])
         : fromToken.value
     ));
+    const showProtocolIcon = computed(() => PROTOCOL_SYMBOLS.includes(imgToken.value?.symbol!));
+    const imgTokenProtocol = computed(() => {
+      if (imgToken.value?.isAe || imgToken.value?.contractId === AE_CONTRACT_ID) {
+        return PROTOCOL_AETERNITY;
+      }
+      if (imgToken.value?.symbol === BTC_SYMBOL) {
+        return PROTOCOL_BITCOIN;
+      }
+      if (imgToken.value?.symbol === ETH_SYMBOL) {
+        return PROTOCOL_ETHEREUM;
+      }
```

Comment:
I strongly recommend to avoid creating list of `if`s or `switch..case` with every protocol.
I see that this component is really messed up with logic that checks the protocols. So maybe we could create a task for refactoring this?